pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -513,14 +720,15 @@ def __init__(self, adapter, name="Hewlett-Packard HP856Xx", **kwargs):
 
         """,
         validator=joined_validators(strict_discrete_set, truncated_discrete_set),
-        values=[["AUTO", "MAN"], [10, 20, 30, 40, 50, 60, 70]]
+        values=[["AUTO", "MAN"], arange(10, 80, 10)],
+        cast=int
```

Comment:
It's good practice to finish the last line with a comma as well. That way, adding more lines (or removing a line) does not change any other line than the added/removed ones.
```suggestion
        cast=int,
```